### PR TITLE
almalinux: Inital support.

### PR DESCRIPTION
**Pull request by avladimirov-storpool**

### CODE REVIEW

File: roles/1_prepare/tasks/check-requirements.yml [hunk 1]
```diff
@@ -9,7 +9,8 @@
         (ansible_distribution == 'Ubuntu' and ansible_distribution_release == 'focal') or
         (ansible_os_family == 'RedHat' and ansible_distribution_major_version == '6') or
         (ansible_os_family == 'RedHat' and ansible_distribution_major_version == '7') or
-        (ansible_os_family == 'RedHat' and ansible_distribution_major_version == '8')
+        (ansible_os_family == 'RedHat' and ansible_distribution_major_version == '8') or
+        (ansible_os_family == 'AlmaLinux' and ansible_distribution_major_version == '8')
     msg: >-
       The only supported platforms for this release are Debian Stretch, Ubuntu 18.04 LTS (Bionic) or
```
Comment (krokodilerian) [inline roles/1_prepare/tasks/check-requirements.yml:15]: can you edit the message also to mention centos 8 and alma? we seem to have missed it

File: roles/2_setup-infra/tasks/subtasks/tune-system-almalinux.yml [hunk 2]
```diff
@@ -0,0 +1,93 @@
+---
```
Comment (krokodilerian) [inline roles/2_setup-infra/tasks/subtasks/tune-system-almalinux.yml:1]: can we reuse the whole thing from c8?

File: roles/3_install-storpool/tasks/subtasks/post_install_almalinux.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (krokodilerian) [inline roles/3_install-storpool/tasks/subtasks/post_install_almalinux.yml:1]: same as the other one, can we reuse c8?

File: roles/1_prepare/tasks/check-requirements.yml (hunk 1 above)
Comment (avladimirov-storpool) [reply roles/1_prepare/tasks/check-requirements.yml]: Done.

File: roles/2_setup-infra/tasks/subtasks/tune-system-almalinux.yml (hunk 2 above)
Comment (avladimirov-storpool) [reply roles/2_setup-infra/tasks/subtasks/tune-system-almalinux.yml:1]: I'm not sure and have to check.

**Comment by netcho** [comment]
It'd be quite hard to write meaningful code without Ansible knowing that AlmaLinux (and Rocky) are part of the Red Hat family. However, this means we need to start using Ansible 2.11 at least

**Pull request closed without merging**